asm/macros/movement.inc

```diff
@@ -1,131 +1,166 @@
-	.macro create_movement_action name
-	enum _\name
+	.macro create_movement_action name, value
```

Comment:
Looks like the old macro was created before we started adding argument requirements.
Should be `.macro create_movement_action name:req, value:req`